### PR TITLE
Addition of rate strategies deployment

### PR DESCRIPTION
**Pull request by eboadom**

### CODE REVIEW

File: scripts/AaveV3EthereumActions.s.sol [hunk 1]
```diff
@@ -26,10 +28,36 @@ contract DeployPayload is Script {
   function run() external {
     vm.startBroadcast();
     IGenericV3ListingEngine v3ListingEngine = IGenericV3ListingEngine(
-      0xc148f4a658105dAD8D1F139b00f99aB914CeDd54
+      0xc148f4a658105dAD8D1F139b00f99aB914CeDd54 // TODO replace with final one of v3 Ethereum
```
Comment (sendra) [inline scripts/AaveV3EthereumActions.s.sol:31]: this provably should go to address book once its deployed then use from there

File: src/contracts/AaveV3EthereumConfigsLib.sol [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  }
+
+  // For LINK, WBTC (and AAVE, but not enabled to borrow)
+  function _rateVolatilesOne() internal pure returns (RateStrategyConfig memory) {
+    return
+      RateStrategyConfig({
+        optimalUsageRatio: _bpsToRay(45_00),
+        baseVariableBorrowRate: 0,
+        variableRateSlope1: _bpsToRay(7_00),
+        variableRateSlope2: _bpsToRay(30_00),
```
Comment (kyzia551) [inline src/contracts/AaveV3EthereumConfigsLib.sol]: I think that it's incorrect, it should be 300_00, based on https://github.com/aave/aave-v3-deploy/blob/main/markets/aave/rateStrategies.ts#L9

File: src/contracts/AaveV3EthereumConfigsLib.sol [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(30_00),
```
Comment (kyzia551) [inline src/contracts/AaveV3EthereumConfigsLib.sol]: Same 300_00

**Review by kyzia551 — CHANGES_REQUESTED** [review]

File: scripts/AaveV3EthereumActions.s.sol (hunk 1 above)
Comment (eboadom) [reply scripts/AaveV3EthereumActions.s.sol:31]: Agree. Will deploy during today the engine from aave-helpers, then we can move to address book

**Review by kyzia551 — APPROVED** [review]

**Pull request merged**